test/Microsoft.DotNet.ToolPackage.Tests/SampleGlobalToolWithShim/consoledemo.csproj

```diff
@@ -3,5 +3,9 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
      <AssemblyName>consoledemo</AssemblyName>
+     <RestoreSources>
+      $(RestoreSources);
+      https://api.nuget.org/v3/index.json;
+     </RestoreSources>
```

Comment:
Why is this needed?

Reply:
Because this project needs to be moved to test assets, but this change was already too big and I wanted to refrain from it. This feed is needed otherwise it fails to find some dependencies, rolls them forward and fails the test.